Functions.Templates/Templates-v2/EventHubTrigger-Python/blueprint_body.py

```diff
@@ -0,0 +1,6 @@
+
+@$(BLUEPRINT_FILE_NAME).event_hub_message_trigger(arg_name="azeventhub", event_hub_name="$(EVENTHUB_NAME)",

```

Comment:
There is no token `$(BLUEPRINT_FILE_NAME)` defined in template.json - did you mean `$(BLUEPRINT_FILENAME)`?
```suggestion
@$(BLUEPRINT_FILENAME).event_hub_message_trigger(arg_name="azeventhub", event_hub_name="$(EVENTHUB_NAME)",
```